packages/nc-gui-v2/components/smartsheet/sidebar/index.vue

```diff
@@ -32,12 +34,19 @@ let viewCreateTitle = $ref('')
 /** is view creation modal open */
 let modalOpen = $ref(false)
 
-/** Watch current views and on change set the next active view */
+/** Watch route param and change active view based on `viewTitle` */
 watch(
-  views,
-  (nextViews) => {
-    if (nextViews.length) {
-      activeView.value = nextViews[0]
+  [views, () => route.params.viewTitle],
+  ([views, viewTitle]) => {
```

Comment:
This is sort of a naming conflict, use `nextViews` instead of the same name as the already existing variable `views`